packages/cli/src/commands/doctor/healthchecks/iosDeploy.ts

```diff
@@ -1,58 +1,116 @@
 import execa from 'execa';
+import chalk from 'chalk';
+// @ts-ignore untyped
+import inquirer from 'inquirer';
 import {checkSoftwareInstalled, PACKAGE_MANAGERS} from '../checkInstallation';
 import {packageManager} from './packageManagers';
-import {logManualInstallation} from './common';
+import {logManualInstallation, removeMessage} from './common';
 import {HealthCheckInterface} from '../types';
+import ora from 'ora';
 
-const getInstallationCommand = () => {
+const label = 'ios-deploy';
+
+const installationWithYarn = 'yarn global add ios-deploy';
+const installationWithNpm = 'npm install ios-deploy --global';
+
+const identifyInstallationCommand = () => {
   if (packageManager === PACKAGE_MANAGERS.YARN) {
-    return 'yarn global add ios-deploy';
+    return installationWithYarn;
   }
 
   if (packageManager === PACKAGE_MANAGERS.NPM) {
-    return 'npm install ios-deploy --global';
+    return installationWithNpm;
   }
 
   return undefined;
 };
 
+const installLibrary = async ({
```

Comment:
Do you mean in the sense that it can be shared?

I would keep it like this until it actually has the need to move to a `common` file, to be clear I would wait until we have another health check with a library from npm registry to move this so we know better what should be kept or changed.

Reply:
Sure! That sounds good. It's ready to be shared when needed 😄